lib/menu.rb

```diff
@@ -0,0 +1,39 @@
+require 'order'
+require 'takeaway'
+
+class Menu
+  # Dish class here is like Menu of dishes
+  attr_reader :dishes
+
+  def initialize(dishes = [{ name: 'chinese', price: 8 }, 
```

Comment:
What do you think about not initializing a class with a data structure? It creates a hard dependency on that data structure